src/experimental/safe_html_transform/browser/configure.zcml

```diff
@@ -17,4 +17,15 @@
         directory="static"
         />
 
+    <adapter factory=".controlpanel.FilterControlPanelAdapter" />
```

Comment:
@prakharjoshi I think you can remove the FilterControlPanelAdapter. This is required to keep backwards compatibility in Plone core only. We don't need it in an add-on product. Remove that line and the FilterControlPanelAdapter class in controlpanel.py as well.